### DISCUSSION TITLE
Number of hydrogen bonds

### DISCUSSION THREAD

**Discussion by horlust** [MDAnalysis/mdanalysis#4690] [Q&A]
I'm studying hydrogen bonds in a water system. The commonly accepted parameters are: D-A distance = 3.5 Å and H-D-A angle = 30°. To analyze the hydrogen bonds, I plotted the number of hydrogen bonds per frame using GROMACS, MDAnalysis, and VMD.

For MDAnalysis, I used an angle of 150°, as it represents the D-H-A angle. However, I observed significant discrepancies in the results across these programs, as shown in the plot below:

![image](https://github.com/user-attachments/assets/cb878d65-35fb-403d-bbb7-b05d1c45d117)

Given these differences, I am concerned about whether I might be doing something wrong or if there might be an issue with the methodology. Why is it so different?

Thank you in advance!

**Comment by orbeckst** [comment]
Do the different programs define donors and acceptors in the same way?

Did you try an example where you know the answer, e.g., some carefully crafted structure with a few molecules?
